code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2472,6 +2472,56 @@
 		T.health -= 20
 		T.mutation_mod += 0.1
 
+/datum/reagent/toxin/insecticide
+	name = "Insecticide"
+	id = INSECTICIDE
+	description = "A broad pesticide. Do not ingest!"
+	reagent_state = REAGENT_STATE_LIQUID
+	color = "#49002E" //rgb: 73, 0, 46
+	density = 1.08
+	specheatcap = 4.18
+
+/datum/reagent/toxin/insecticide/reaction_mob(var/mob/living/M, var/method = TOUCH, var/volume)
+
+	if(..())
+		return 1
+	if(iscarbon(M))
+		var/mob/living/carbon/C = M
+		if(!C.wear_mask) //If not wearing a mask
+			C.adjustToxLoss(REM) //4 toxic damage per application, doubled for some reason
+		if(isinsectoid(C) || istype(C, /mob/living/carbon/monkey/roach)) //Insecticide being poisonous to bugmen, who'd've thunk
+			M.adjustToxLoss(10 * REM)
+					
+/datum/reagent/toxin/insecticide/reaction_animal(var/mob/living/simple_animal/M, var/method=TOUCH, var/volume)
+	if(..())
+		return 1
+		
+	if(istype(M,/mob/living/simple_animal/hostile/bigroach))
+		M.gib()
+		
+	if(istype(M,/mob/living/simple_animal/cockroach))
+		M.gib()
+	
+	if(istype(M,/mob/living/simple_animal/bee))
+		M.gib()
+
+	if(istype(M,/mob/living/simple_animal/hostile/giant_spider))
```

Comment:
Instagibbing all spiders, including queens, makes them completely trivial as a threat, not thought through; just get rid of it

Reply:
I didn't? Fuck me, I didn't notice. I was trying for it to instagib the small spiders. Again, jury rigged.

Reply:
Do what I said above regarding reagent_act() inside `simple_animal/hostile/giant_spider/spiderling.dm`; unlike the other 2, it doesn't have an existing proc, so you need to make one -- make sure when you path it that it includes the full path (e.g.: `mob/living/simple_animal/hostile/giant_spider/spiderling/reagent_act(id, method, volume)` )

Reply:
Fixed, I think.